code/game/jobs/job/assistant.dm

```diff
@@ -28,6 +28,10 @@
 	H.equip_or_collect(new /obj/item/clothing/under/color/grey(H), slot_w_uniform)
 	H.equip_or_collect(new /obj/item/clothing/shoes/black(H), slot_shoes)
 	H.equip_or_collect(new H.species.survival_gear(H.back), slot_in_backpack)
+	if(prob(1))
+		H.equip_or_collect(new /obj/item/weapon/storage/bag/plasticbag(H), slot_head)
```

Comment:
While funny, I don't think we should do this for the designated new player job. People might suffocate before learning the controls.